docs/data/material/getting-started/templates/.eslintrc.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
-  rules: {
-    'jsx-a11y/anchor-is-valid': 'off',
-  },
-};
```

Comment:
Initially, I was planning on moving this to the root per https://github.com/mui/mui-public/issues/213.

But then realize that this shouldn't be here in the first place. Developers will get the same experience once they copy and paste this into their filesystem.